src/wasm-lib/kcl/src/executor.rs

```diff
@@ -1438,6 +1438,19 @@ pub enum Path {
         /// arc's direction
         ccw: bool,
     },
+    /// a complete arc
+    Circle {
```

Comment:
Sorry if this is obvious, but, why is `Circle` a kind of path segment? How can you add more path segments before or after a circle? It doesn't make sense to me to have a circle as part of a sequence of paths. Shouldn't it just be a shortcut for creating a path with 1 segment, a TangentialArc from 0 to 360 degrees?

Reply:
inner_arc is very messed up because some of the arc structs are messed up, it returns a path segment using `ToPoint`, I didn't want to change how the arc segments work in case that's a breaking change. I know if the other arcs are not working correctly I'm just leaving the dirt under the rug and not doing anything about it, but I plan on taking a second pass where I add arcs to the client side scene in which case the arcs structs in rust will probably need some attention too, in which case I'll likely remove this struct again.

I was talking to @jtran about it too, I'll likely remove all but one of the arcs, I don't think there's a need for tangencialArc and arc for example, I think we got confused between the API for users and the functions available to them vs the data we store under-the-hood, I think we only need a struct for each curve type, so straight lines and arcs at this point, but later splines etc etc.

Do you think it's okay to put this circle struct in here short term?

Reply:
Yeah in the interests of getting this merged I'm happy to include it, but it doesn't really make sense. Let's consolidate this when you fix up the arcs though. Could you just make an issue for it, and leave the issue in the source code here as a comment? So we don't lose track of it.

Reply:
Sounds good 414b11d6e02a58974a48063da3c37bec5ab6ad8a